config/hydroshare-config.yaml

```diff
@@ -1,18 +1,18 @@
-### Local Configuration Variables ###
-HS_PATH: /home/hydro/hydroshare
-HOST_SSL_DIR: /home/hydro/hs-certs
-HS_DATABASE: pg.development.sql
-HS_LOG_FILES: /home/hydro/hydroshare/log
+ ### Local Configuration Variables ###
+ HS_PATH: /home/hydro/hydroshare
+ HOST_SSL_DIR: /home/hydro/hs-certs
+ HS_DATABASE: pg.www.sql
```

Comment:
please change it back to pg.development.sql for default settings. pg.www.sql is just for your testing convenience. In fact, I don't see any valid change in this hydroshare-config.yaml file, so please revert it back to the original version on develop in this branch before it can be merged.